build/squire-raw.js

```diff
@@ -3576,7 +3576,11 @@ if ( top !== win ) {
         win.onEditorLoad = null;
     }
 } else {
-    win.Squire = Squire;
+    if ( module && module.exports ) {
```

Comment:
Will be using this pattern for checking if module is there
https://github.com/umdjs/umd/blob/master/returnExports.js#L45